autodoc/data/admin-api.lua

```diff
@@ -903,8 +902,7 @@ return {
         ]] },
         protocols = {
           description = [[
-            A list of the request protocols that will trigger this plugin. Possible values are
-            `"http"`, `"https"`, `"tcp"`, and `"tls"`.
+            A list of the request protocols that will trigger this plugin.
 
             The default value, as well as the possible values allowed on this field, may change
             depending on the plugin type. For example, plugins that only work in stream mode will
```

Comment:
Unrelated: "For example, plugins that only work in stream mode `will may only` support "tcp" and "tls"." -> "For example, plugins that only work in stream mode `will only` support "tcp" and "tls"."

Reply:
sneaked the fix in this PR as well! thanks!